dartagnan/src/main/java/com/dat3m/dartagnan/program/event/lang/svcomp/LoopBound.java

```diff
@@ -0,0 +1,30 @@
+package com.dat3m.dartagnan.program.event.lang.svcomp;
+
+import com.dat3m.dartagnan.program.event.core.annotations.FunCall;
+
+public class LoopBound extends FunCall {
```

Comment:
I'm fine with changing that. But as you said, because these are also function calls at the C-level and the other annotations where `FunCall`s, I kept it initially.

Reply:
Every single annotation we could potentially have will result in a function call at the C level. If we follow this idea, the nothing is really an annotation. I think it makes much more sense to use FunCall for real C functions

Reply:
I agree with that. I will remove the `FunCall` inheritance.